spec/models/conference_spec.rb

```diff
@@ -41,6 +41,16 @@
       end
     end
   end
+  
+  describe '#dates' do
+    subject { FactoryGirl.build_stubbed(:conference) }
+    
+    it 'has a date range' do
```

Comment:
Hm, ok.

So the last expectation on its own should work. So if you remove the first one,
```ruby
it "bla bla" do
  expect(subject.dates).to be_a(DateRange)
end
```
this should pass.

For the rest, maybe you have to add `and_call_original``
```ruby
it "bla bla" do
  expect(DateRange)
    .to receive(:new)
    .with(start_date: subject.starts_on, end_date: subject.ends_on)
    .and_call_original
  # etc.
end
```